source/catalog.html

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body>
+  <header class="page-header">
+    <a class="page-header__logo">
+      <img class="page-header__logo-image" src="img/logo_mobile.svg" width="191" height="39" alt="CAT ENERGY">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__open-menu" type="button">Открыть меню</button>
+      <ul class="main-nav__list site-list">
+        <li class="site-list__item">
+          <a href="index.html">Главная</a>
+        </li>
+        <li class="site-list__item site-list__item--active">
+          <a>Каталог продукции</a>
+        </li>
+        <li class="site-list__item">
+          <a href="form.html">Подбор программы</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="page-main">
+    <h1 class="page-main__head">Каталог продукции</h1>
+    <section class="page-main__catalog">
+      <ul class="page-main__catalog-list catalog">
+        <li class="catalog__item product product--chicken-500">
+          <a href="#" class="product__link">
+            <img class="product__image" src="img/catalog-1-mobile.jpg" alt="Cat Energy PRO 500 г" width="140"
+              height="144">
+          </a>
+          <h2 class="product__title">Cat Energy PRO 500 г</h2>
+          <table class="product__properties">
+            <tr>
+              <td class="product__field">
+                Объем
+              </td>
+              <td class="product__value">
+                500 г
+              </td>
+            </tr>
+            <tr>
+              <td class="product__field">
+                Вкус
+              </td>
+              <td class="product__value">
+                Курица
+              </td>
+            </tr>
+            <tr>
+              <td class="product__field">
+                Цена
+              </td>
+              <td class="product__value">
+                700 Р.
+              </td>
+            </tr>
+          </table>
+          <a href="#" class="product__order button">Заказать</a>
+        </li>
+        <li class="catalog__item product product--chicken-1000">
+          <a href="#" class="catalo__link">
```

Comment:
```suggestion
          <a href="#" class="catalog__link">
```

поправь все вхождения